virttest/qemu_monitor.py

```diff
@@ -2214,7 +2214,8 @@ def send_args_cmd(self, cmdlines, timeout=CMD_TIMEOUT, convert=True):
                     value = "=".join(opt[1:])
                     try:
                         if re.match("^[0-9]+$", value):
-                            value = int(value)
+                            if not (opt[0] == 'fd'):
+                                value = int(value)
```

Comment:
What's the use case for it? If it is all numbers, what's the difference of 'int' or 'str' here?

Reply:
Ah, okay I see the product bz, so it can't be int in new released qemu.

Reply:
@vivianQizhu qemu 5.0 only accept str now, and str can work with old version too include rhel7.
You can get more details from the bz i added.